### DISCUSSION TITLE
How to calculate intersection area of two rotated rectangles?

### DISCUSSION THREAD

**Discussion by Boltzmachine** [georust/geo#1138] [Q&A]
I would like to calculate the intersection area of two rectangles of any orientation. But I cannot figure out how I can achieve this by reading the documentation. Can anyone help me?

**Comment by lnicola** [comment]
I guess there are a couple of parts to this question:

 - Q: how do I represent a rotated rectangle? A: as a `geo_types::Polygon`
 - Q: how do I rotate a geometry? A: using `geo::Rotate`
 - Q: how do I intersect two geometries? A: using `geo::BooleanOps`
 - Q: how do I compute the area of a geometry? A: using `geo::Area`

If you have other questions, don't hesitate to ask on Discord or GitHub Discussions!

**Comment by Boltzmachine** [comment]
Thanks. I will try
